src/main/java/org/spdx/sbom/gradle/utils/SpdxListedLicenseEmbeddedStore.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2024 The Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.spdx.sbom.gradle.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import org.spdx.library.InvalidSPDXAnalysisException;
+import org.spdx.storage.listedlicense.SpdxListedLicenseModelStore;
+
+/**
+ * @author Gary O'Neall
```

Comment:
cc @goneall

Reply:
Oh, this is cause this file was just copied from the spdx-java-library project, so kept the original author intact

Reply:
maybe I should add a note to this comment

Reply:
updated the comment